pelicanconf.py

```diff
@@ -1,3 +1,27 @@
 # -*- coding: utf-8 -*-
 
+AUTHOR = 'Komitywa'
+SITENAME = 'Komitywa.org'
+SITEURL = ''
+
+THEME = 'themes/notmyidea'
+
+PATH = 'content'
```

Comment:
If we want to define `PATH` here, pointing content directory should be removed from `.travis.yml`, `fabfile.py` and `README.md` should be updated also.